packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -672,10 +676,11 @@ async function handler(
 
   const fullName = getFullName(bookerName);
 
+  // Why are we only using "en" locale
   const tGuests = await getTranslation("en", "common");
 
   const dynamicUserList = Array.isArray(reqBody.user) ? reqBody.user : getUsernameList(reqBody.user);
-  if (!eventType) throw new HttpError({ statusCode: 404, message: "eventType.notFound" });
+  if (!eventType) throw new HttpError({ statusCode: 404, message: "EventType not Found" });
```

Comment:
`question-non-blocking`: Should this go in `ErrorCodes`?

Reply:
yeah I'll add this